doc/01-installation.md

```diff
@@ -3,7 +3,7 @@
 ## Installation
 
 > **Note**
-> Please note that castor needs PHP >= 8.1 to run.
+> Castor needs PHP >= 8.1 to run.
```

Comment:
Your repo was outdated?

Reply:
Nope, it was fixed on the readme but not on the doc